doc/reference/reference_rock/vshard/vshard_ref.rst

```diff
@@ -214,7 +214,7 @@ Router public API
 * :ref:`vshard.router.callbre(bucket_id, function_name, {argument_list}, {options}) <router_api-callbre>`
 * :ref:`vshard.router.route(bucket_id) <router_api-route>`
 * :ref:`vshard.router.routeall() <router_api-routeall>`
-* :ref:`vshard.router.bucket_id_(key) <router_api-bucket_id_strcrc32>`
+* :ref:`vshard.router.bucket_id_strcrc32(key) <router_api-bucket_id_strcrc32>`
```

Comment:
Since you added bucket_id_strcrc32 in this list, Probably the mp version also should be mentioned.